slides/src/main/asciidoc/lecture.adoc

```diff
@@ -237,7 +249,7 @@ A1 -> B1 -> B2 ->A2 ;
 
 A2->Kafka;
 
-Kafka[shape=box,label="reduce \n (a, b) → <STOP>"]
+Kafka[shape=box,label="reduce \n (l, r) → <STOP>"]
```

Comment:
I thought it was confusing to use `a, b` for the reduce arguments, since our records are `A, B`, but the reduce arguments don't refer to these different keys; rather subsequent values for the same key. `l, r` refers to "left" and "right"; not sure if that's clear.